Gemfile.lock

```diff
@@ -867,11 +855,6 @@ GEM
     view_component (2.47.0)
       activesupport (>= 5.0.0, < 8.0)
       method_source (~> 1.0)
-    virtus (1.0.5)
```

Comment:
ooh, is this the last dependency on virtus? 

I assume all the other gems moved to dry.rb gems, since the author of virtus openly deprecated it in favor of  dry-types, or rolled their own implementations if they needed it.

Reply:
When I saw that we were removing Virtus, I was quite happy.  I love Virtus but DryRB is so much nicer (and maintained).

Reply:
This PR ends up removing 6 gems in total, which I really like. Going through the Rails 7 update process once again confirms that every dependency is a liability. I mean even revisit usage of `store_attribute` for profile fields at one point, as the scope changed we don't have much need for typed accessors as we're only dealing with strings now.